cypress/integration/reducer.unit.test.js

```diff
@@ -1,29 +1,31 @@
-import { storeEdges, selectNode, clearSelection } from '../../../src/visualizer/actions/viewport.ts';
-import reducer from '../../../src/visualizer/reducers/index';
+import { storeEdges, selectNode, clearSelection } from '../../src/visualizer/actions/viewport.ts';
```

Comment:
Per conversation with Sean, I need to work on this section once the PR is approved. Need to fix some of the syntax here